### DISCUSSION TITLE
Month language

### DISCUSSION THREAD

**Discussion by morganesoula** [hi-manshu/Kalendar#62] [Q&A] [answered]
Hey there, do you know if it is possible to change the language of the month title?

**Comment by gsrathoreniks** [comment, marked as answer]
@morganesoula No, right now it only supports English. 
You're looking for support for which language?

**Reply by morganesoula** [reply]
French but it would be nice if it was according to the device default language

**Reply by gsrathoreniks** [reply]
Got it, will discuss and if all good will add support for popular languages.
